marketplace_builder/views/pages/reference/form-configurations/validation.liquid

```diff
@@ -0,0 +1,246 @@
+</code></pre>
+### length
+{% raw %}
+<pre><code class="language-yaml">
+    attachments:
+      validation:
+        length:
+          is: 2
+          message: "You need to include Front side and Back side Photo ID"
+</code></pre>
+{% endraw %}
+
+The possible length constraint options are:
+
+* :minimum - The attribute cannot have less than the specified length.
+* :maximum - The attribute cannot have more than the specified length.
+* :in (or :within) - The attribute length must be included in a given interval. The value for this option must be a range.
+* :is - The attribute length must be equal to the given value.
+
+
+### numericality
+
+{% raw %}
+<pre><code class="language-yaml">
+            rate_amount:
+              validation:
+                presence: true
+                numericality:
+                  greater_than_or_equal_to: 0
+</code></pre>
+{% endraw %}
+
+Available options are:
+
+* only_integer
+* greater_than - Specifies the value must be greater than the supplied value. The default error message for this option is "must be greater than %{count}".
+* greater_than_or_equal_to - Specifies the value must be greater than or equal to the supplied value. The default error message for this option is "must be greater than or equal to %{count}".
+* equal_to - Specifies the value must be equal to the supplied value. The default error message for this option is "must be equal to %{count}".
+* less_than - Specifies the value must be less than the supplied value. The default error message for this option is "must be less than %{count}".
+* less_than_or_equal_to - Specifies the value must be less than or equal to the supplied value. The default error message for this option is "must be less than or equal to %{count}".
+* other_than - Specifies the value must be other than the supplied value. The default error message for this option is "must be other than %{count}".
+* odd - Specifies the value must be an odd number if set to true. The default error message for this option is "must be odd".
+* even - Specifies the value must be an even number if set to true. The default error message for this option is "must be even".
+
+## Conditional Validation
+
+It is possible to validate attribute based on the state of other attribute from the form. 
+{% raw %}
+<pre><code class="language-yaml">
+            expiry_year:
+              validation:
+                presence:
+                  if: '{% if form.expires %}true{% endif %}'
+</code></pre>
+{% endraw %}
+
+
+## Common Validation Options
+
+### message
+It is possible to overwrite the default message with additional configuration:
+
+{% raw %}
+<pre><code class="language-yaml">
+      customizations:
+        expertise:
+          validation:
+            presence:
+              message: 'record is required.'
+</code></pre>
+{% endraw %}
+
+### allow_nil
+The ```allow_nil``` option skips the validation when the value being validated is nil.
```

Comment:
tripe backtick is not supported in our markdown - use single backtick to enable inline <code> block